finagle-mysql/src/test/scala/com/twitter/finagle/mysql/unit/RequestTest.scala

```diff
@@ -2,6 +2,7 @@ package com.twitter.finagle.exp.mysql
 
 import java.sql.{Timestamp, Date => SQLDate}
 import java.util.{Calendar, Date, TimeZone}
+import com.twitter.finagle.exp.mysql.Parameter.NullParameter
```

Comment:
@mosesn Tried removing that line and re-auto-importing. For some reason, IDEA keeps putting it on the same line! Will move it manually and push.